src/components/Books.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Books = () => (
+  <>
+    <ListBooks />
+    <AddBooks />
+  </>
+);
+
+const ListBooks = () => {
+  const booksData = [{
+    id: 1,
+    name: 'The Hunger Games',
+    author: 'Suzanne Collins',
+  }, {
+    id: 2,
+    name: 'Dune',
+    author: 'Frank Herbert',
+  }, {
+    id: 3,
+    name: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+  }];
+
+  return (
+    <ul className="books_list">
+      {booksData.map((data) => (
+        <li key={data.id}>
+          <Book name={data.name} author={data.author} />
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const Book = (props) => {
+  const { name, author } = props;
+  return (
+    <article>
+      <h2 className="name">{name}</h2>
+      <p className="author text_style_6">{author}</p>
+      <span>
+        <Comments />
+        <Remove />
+        <Edit />
+      </span>
+    </article>
+  );
+};
+
+const Comments = () => <button type="button" className="text_style_6">Comments</button>;
+
+const Remove = () => {
+  const removeBook = () => {
+
+  };
+  return <button type="button" className="text_style_6" onClick={removeBook}>Remove</button>;
+};
```

Comment:
- [ ] As mentioned above, it would be nice to move these functions into a different file to keep your codebase more organized. So please apply this to all the functions that may need it.

Reply:
Thanks for the tip. Have a nice day.